otel.yml

```diff
@@ -21,6 +21,9 @@ extensions:
     check_interval: 1s
     limit_mib: 700
     spike_limit_mib: 180
+  health_check:
+    endpoint: "localhost:13133"
```

Comment:
I chose this value based on the documentation in https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/main/extension/healthcheckextension/README.md#health-check.  I'm happy to change it to another value if others have a better suggestion.